backend/src/schema/resolvers/searches/queryString.js

```diff
@@ -0,0 +1,53 @@
+export function queryString(str) {
+  const normalizedString = normalizeWhitespace(str)
+  const escapedString = escapeSpecialCharacters(normalizedString)
+  return `
+${matchWholeText(escapedString)}
+${matchEachWordExactly(escapedString)}
+${matchSomeWordsExactly(escapedString)}
+${matchBeginningOfWords(escapedString)}
+`
+}
+
+const matchWholeText = (str, boost = 8) => {
+  return `"${str}"^${boost}`
```

Comment:
thank you for giving the docs so that your reviewers can understand how the boost works!
I didn't really understand how you came to the defaults. Maybe you could explain?

Reply:
I thought: 1, 2, 4, 8
Next would be 16
Matching the whole query exactly -> 8
Matching all words exactly -> 4
Matching at least one word exactly -> 2
Match the beginning of some words -> 1